exasol_integration_test_docker_environment/cli/commands/spawn_test_environment.py

```diff
@@ -45,6 +45,13 @@
     show_default=True,
     help="Host port to which the BucketFS port gets forwarded",
 )
+@click.option(
+    "--ssh-port-forward",
+    type=int,
+    default=None,
+    show_default=True,
+    help="Host port to which the SSH port gets forwarded",
```

Comment:
if not selected, one will be randomly selected

Reply:
yes, this was the idea

Reply:
Understood thanks for explanation, will be implemented.